templates/compose/appflowy.yaml

```
@@ -0,0 +1,217 @@
+# slogan: AppFlowy is the AI collaborative workspace where you achieve more without losing control of your data.
+# tags: appflowy,notion,open-source,productivity,notes,ai,self-hosted
+# logo: svgs/appflowy.svg
+# category: productivity
+# port: 80
+
+services:
+  appflowy_web:
+    environment:
+      - SERVICE_FQDN_APPFLOWYWEB_80
+      - APPFLOWY_BASE_URL=${SERVICE_FQDN_APPFLOWYCLOUD_8000}
+      - APPFLOWY_GOTRUE_BASE_URL=${SERVICE_FQDN_GOTRUE_9999}
+      - APPFLOWY_WS_BASE_URL=${SERVICE_FQDN_APPFLOWYCLOUD_8000}/ws/v2
+    image: appflowyinc/appflowy_web:latest
+    depends_on:
+      - appflowy_cloud
+
+  appflowy_cloud:
+    image: appflowyinc/appflowy_cloud:latest
+    labels:
+      - "traefik.http.middlewares.appflowy-cors.headers.customRequestHeaders.X-Request-Id={{.RequestID}}"
+      - "traefik.http.middlewares.appflowy-cors.headers.accessControlAllowMethods=GET,POST,PUT,DELETE,PATCH,OPTIONS"
+      - "traefik.http.middlewares.appflowy-cors.headers.accessControlAllowHeaders=Content-Type,Authorization,Accept,Client-Version,Device-Id"
+      - "traefik.http.middlewares.appflowy-cors.headers.accessControlMaxAge=3600"
+      - "traefik.http.middlewares.appflowy-cors.headers.accessControlAllowOriginList=*"
+      - "traefik.http.middlewares.appflowy-cors.headers.addvaryheader=true"
+    environment:
+      - SERVICE_FQDN_APPFLOWYCLOUD_8000
+      - RUST_LOG=${RUST_LOG:-info}
+      - APPFLOWY_ENVIRONMENT=production
+      - APPFLOWY_DATABASE_URL=postgres://${SERVICE_USER_POSTGRES}:${SERVICE_PASSWORD_POSTGRES}@postgres:5432/${POSTGRES_DB:-appflowy}
+      - APPFLOWY_REDIS_URI=redis://redis:6379
+      - APPFLOWY_GOTRUE_JWT_SECRET=${SERVICE_PASSWORD_JWT}
+      - APPFLOWY_GOTRUE_JWT_EXP=${GOTRUE_JWT_EXP:-7200}
+      - APPFLOWY_GOTRUE_BASE_URL=http://gotrue:9999
+      - APPFLOWY_S3_CREATE_BUCKET=${APPFLOWY_S3_CREATE_BUCKET:-true}
+      - APPFLOWY_S3_USE_MINIO=true
+      - APPFLOWY_S3_MINIO_URL=http://minio:9000
+      - APPFLOWY_S3_ACCESS_KEY=${SERVICE_USER_MINIO}
+      - APPFLOWY_S3_SECRET_KEY=${SERVICE_PASSWORD_MINIO}
+      - APPFLOWY_S3_BUCKET=${APPFLOWY_S3_BUCKET:-appflowy}
+      - APPFLOWY_S3_REGION=${APPFLOWY_S3_REGION:-us-east-1}
+      - APPFLOWY_MAILER_SMTP_HOST=${APPFLOWY_MAILER_SMTP_HOST:-smtp.gmail.com}
+      - APPFLOWY_MAILER_SMTP_PORT=${APPFLOWY_MAILER_SMTP_PORT:-587}
+      - APPFLOWY_MAILER_SMTP_USERNAME=${APPFLOWY_MAILER_SMTP_USERNAME:-notify@appflowy.io}
+      - APPFLOWY_MAILER_SMTP_EMAIL=${APPFLOWY_MAILER_SMTP_EMAIL:-notify@appflowy.io}
+      - APPFLOWY_MAILER_SMTP_PASSWORD=${APPFLOWY_MAILER_SMTP_PASSWORD:-email_sender_password}
+      - APPFLOWY_MAILER_SMTP_TLS_KIND=${APPFLOWY_MAILER_SMTP_TLS_KIND:-none}
+      - APPFLOWY_ACCESS_CONTROL=true
+      - APPFLOWY_DATABASE_MAX_CONNECTIONS=40
+      - AI_SERVER_HOST=ai
+      - AI_SERVER_PORT=5001
+      - AI_OPENAI_API_KEY=${AI_OPENAI_API_KEY:-}
+      - APPFLOWY_WEB_URL=${SERVICE_FQDN_APPFLOWYWEB_80}
```

Since the latest update a few days ago, all `SERVICE_FQDN` and `SERVICE_URL` now work in reverse and need to be swapped.
```suggestion
      - APPFLOWY_WEB_URL=${SERVICE_URL_APPFLOWYWEB_80}
```

---

fixed